### PR TITLE
Update HB witherspoon config to use MEMVPD config flag instead of CVPD

### PR DESCRIPTION
**Pull request by pragupta**
This change is [Reviewable](https://reviewable.io/reviews/open-power/op-build/849)

### CODE REVIEW

**Comment by wghoffa** [comment]
@williamspatrick - I think this is ready to go. Can you merge this?

File: openpower/configs/hostboot/witherspoon.config [hunk 1]
```diff
@@ -52,15 +52,15 @@ unset PNOR_TWO_SIDE_SUPPORT
...
 unset ENABLE_CHECKSTOP_ANALYSIS
 unset IPLTIME_CHECKSTOP_ANALYSIS
 
-# Hostboot will detect hardware changes
+# Hostboot will not detect hardware changes
 unset HOST_HCDB_SUPPORT
 
 # set for trace debug to console
-unset CONSOLE_OUTPUT_TRACE
+set CONSOLE_OUTPUT_TRACE
```
Comment (stewart-ibm) [inline openpower/configs/hostboot/witherspoon.config]: Separate patch?

Is this required/how people are currently booting? Or is it just a convenience for hostboot development?

File: openpower/configs/hostboot/witherspoon.config [hunk 2]
```diff
@@ -12,15 +12,15 @@ set MVPD_WRITE_TO_HW
 set MVPD_READ_FROM_PNOR
 set MVPD_WRITE_TO_PNOR
 set DJVPD_READ_FROM_HW
-unset DJVPD_WRITE_TO_HW
+set DJVPD_WRITE_TO_HW
```
Comment (stewart-ibm) [inline openpower/configs/hostboot/witherspoon.config]: Is this part of the same change?

File: openpower/configs/hostboot/witherspoon.config [hunk 3]
```diff
@@ -29,7 +29,7 @@ unset CDIMM_FORMAT_FOR_CVPD
 set GPIODD
 unset PALMETTO_VDDR
 
-# Enable SBE updates
+# Disable SBE updates
```
Comment (stewart-ibm) [inline openpower/configs/hostboot/witherspoon.config]: Please put in a separate patch, this has nothing to do with MEMVPD configuration, it's correcting a comment.

File: openpower/configs/hostboot/witherspoon.config [hunk 4]
```diff
@@ -52,15 +52,15 @@ unset PNOR_TWO_SIDE_SUPPORT
 
 unset BMC_BT_LPC_IPMI
 
-# Enable Checktop Analysis
+# Disable Checktop Analysis
```
Comment (stewart-ibm) [inline openpower/configs/hostboot/witherspoon.config]: Please put in a separate patch, this has nothing to do with MEMVPD configuration, it's correcting a comment.

File: openpower/configs/hostboot/witherspoon.config [hunk 5]
```diff
@@ -52,15 +52,15 @@ unset PNOR_TWO_SIDE_SUPPORT
...
 unset ENABLE_CHECKSTOP_ANALYSIS
 unset IPLTIME_CHECKSTOP_ANALYSIS
 
-# Hostboot will detect hardware changes
+# Hostboot will not detect hardware changes
```
Comment (stewart-ibm) [inline openpower/configs/hostboot/witherspoon.config]: Please put in a separate patch, this has nothing to do with MEMVPD configuration, it's correcting a comment.

File: openpower/configs/hostboot/witherspoon.config (hunk 2 above)
Comment (pragupta) [inline openpower/configs/hostboot/witherspoon.config]: Yes, I updated the commit message to be a bit more clear.

**Comment by williamspatrick** [comment]
@stewart-ibm's comments were resolved.
